astroquery/alma/tests/test_alma_remote.py

```diff
@@ -291,3 +292,50 @@ def test_project_metadata():
     alma = Alma()
     metadata = alma.get_project_metadata('2013.1.00269.S')
     assert metadata == ['Sgr B2, a high-mass molecular cloud in our Galaxy\'s Central Molecular Zone, is the most extreme site of ongoing star formation in the Local Group in terms of its gas content, temperature, and velocity dispersion. If any cloud in our galaxy is analogous to the typical cloud at the universal peak of star formation at z~2, this is it. We propose a 6\'x6\' mosaic in the 3mm window targeting gas thermometer lines, specifically CH3CN and its isotopologues. We will measure the velocity dispersion and temperature of the molecular gas on all scales (0.02 - 12 pc, 0.5" - 5\') within the cloud, which will yield resolved measurements of the Mach number and the sonic scale of the gas. We will assess the relative importance of stellar feedback and turbulence on the star-forming gas, determining how extensive the feedback effects are within an ultradense environment. The observations will provide constraints on the inputs to star formation theories and will determine their applicability in extremely dense, turbulent, and hot regions. Sgr B2 will be used as a testing ground for star formation theories in an environment analogous to high-z starburst clouds in which they must be applied.']
+
+@remote_data
+@pytest.mark.parametrize('dataarchive_url', _test_url_list)
+def test_staging_postfeb2020(dataarchive_url):
+
+    alma = Alma()
+    alma.archive_url = dataarchive_url
+    tbl = alma.stage_data('uid://A001/X121/X4ba')
+
+    assert 'mous_uid' in tbl.colnames
+
+    assert '2013.1.00269.S_uid___A002_X9de499_X3d6c.asdm.sdm.tar' in tbl['name']
+
+
+@remote_data
+@pytest.mark.parametrize('dataarchive_url', _url_list)
+def test_staging_uptofeb2020(dataarchive_url):
+
+    alma = Alma()
+    alma.archive_url = dataarchive_url
+    tbl = alma.stage_data('uid://A001/X121/X4ba')
+
+    assert 'mous_uid' in tbl.colnames
+
+    names = [x.split("/")[-1] for x in tbl[tbl['mous_uid'] == 'uid://A001/X147/X92']['URL']]
+
+    assert '2013.1.00269.S_uid___A002_X9de499_X3d6c.asdm.sdm.tar' in names
+
+
+# always skip this: it's only for testing proprietary access
+@pytest.mark.skipif('True')
```

Comment:
now this is OK, but long term it would be nice to be controlled by an env variable (similar to the slow tests)

Reply:
I like that idea, but I am not sure if that will even work in a pytest environment.  In practice, I had to excise this code from the tests entirely.  I'll remove this test since it's useless.